### PR TITLE
Implement py-car-wash-station

### PR DESCRIPTION
**Pull request by SevKrok**

### CODE REVIEW

**Review by vshvanska — APPROVED** [review]

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,50 @@
...
+            income += self.wash_single_car(car)
+
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            car.comfort_class * self.average_rating
+            * (self.clean_power - car.clean_mark)
+            / self.distance_from_city_center,
+            1)
```
Comment (LiudmylaKulinchenko) [inline app/main.py]: Bad formatting. Better do this:
round(
    ...,
    1
)

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,50 @@
...
+            price = self.calculate_washing_price(car)
+            car.clean_mark = self.clean_power
+
+        return price
+
+    def rate_service(self, mark: int) -> None:
+        self.average_rating = round(
+            (self.average_rating * self.count_of_ratings + mark)
+            / (self.count_of_ratings + 1),
+            1)
```
Comment (LiudmylaKulinchenko) [inline app/main.py]: fix formatting also here

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,50 @@
...
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            car.comfort_class * self.average_rating
+            * (self.clean_power - car.clean_mark)
+            / self.distance_from_city_center,
+            1)
+
+    def wash_single_car(self, car: Car) -> float:
```
Comment (LiudmylaKulinchenko) [inline app/main.py]: No need to use return here

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 3 above)
Comment (SevKrok) [reply app/main.py:37]: But why?
I think this better than use "if" two times and it's contains in one func wash single car

Comment (SevKrok) [reply app/main.py]: ok
understood

**Review by Sel-Fisher — APPROVED** [review]
as always the best and cleanest code i've ever seen

File: app/main.py [hunk 4]
```diff
@@ -1,8 +1,47 @@
...
+                self.wash_single_car(car)
+
+        return income
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            car.comfort_class * self.average_rating
+            * (self.clean_power - car.clean_mark)
+            / self.distance_from_city_center,
+            1)
```
Comment (LiudmylaKulinchenko) [inline app/main.py]:
```suggestion
        return round(
            car.comfort_class * self.average_rating
            * (self.clean_power - car.clean_mark)
            / self.distance_from_city_center, 1
        )
```

fix everywhere

Comment (LiudmylaKulinchenko) [reply app/main.py]: You can find it in the previous task [checklist](https://github.com/mate-academy/py-linter-formatter/blob/master/checklist.md) (at the end of it)

**Review by LiudmylaKulinchenko — CHANGES_REQUESTED** [review]

**Review by fsocie7y — APPROVED** [review]
well done 🚀

**Pull request closed without merging**